go/README.md

```diff
@@ -117,57 +117,6 @@ Status
 The first milestone was to implement the necessary Array types in order to use
 them internally in the [ifql][] execution engine and storage layers of [InfluxDB][].
```

Comment:
There's some other cruft here that ought to be removed. I'll make some additional changes and then merge this